Packages/Tags/Runtime/TagsEventsSub.cs

```diff
@@ -0,0 +1,36 @@
+using UnityEngine;
+using UnityAtoms.Tags;
+
+public class TagEventSub : MonoBehaviour
```

Comment:
This should not be in the repository, this is just for testing purposes, either have it in an example project or create an actual test case in in Packages/Tags/Test not Runtime...

Reply:
I have moved it to Packages/Tags/Test.

Reply:
Thats not what I meant. In the Test assembly, there should be tests, that are actually executable by the unity testrunner.

Your code is a MonoBehaviour, you could write a testcase using that behaviour, or create a PR on the https://github.com/unity-atoms/unity-atoms-examples repository, where you're showing of the usage. 

Or write a page of documentation for the feature, where you show of the usage.

But on it's own, this class should not be in the repository, as it does not provide anything to it.

So, to summarize:
- either remove it from the PR (you can keep it locally, though)
- or create unit tests that make use of the class.

Reply:
No worries. Now that I understood what you meant, I have removed it.